http-agent/src/main/java/org/motechproject/http/agent/domain/HTTPActionRecord.java

```diff
@@ -0,0 +1,88 @@
+package org.motechproject.http.agent.domain;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * The <code>HTTPActionRecord</code> class represents a record of a http Action.
+ */
+@Entity
+public class HTTPActionRecord {
+
+    @Field
+    private String url;
+
+    @Field
+    private String request;
+
+    @Field
+    private String responseBody;
+
+    @Field
+    private String responseCode;
+
+    public HTTPActionRecord(String url, String request, String responseBody, String responseCode) {
+        this.url = url;
+        this.request = request;
+        this.responseBody = responseBody;
+        this.responseCode = responseCode;
+    }
+
+    public void setUrl()  { this.url = url; }
+
+    public void setRequesrt()  { this.request = request; }
```

Comment:
Place the content of the method in the new line. Same with closing brace. http://docs.motechproject.org/en/latest/development/coding_conventions.html#braces